L1Trigger/L1TMuonEndCap/interface/TrackTools.h

```diff
@@ -126,14 +125,29 @@ namespace emtf {
     return rad_to_deg(calc_theta_rad(eta));
   }
 
-  inline int    calc_theta_int(double theta, int endcap, int bits) {  // theta in deg, endcap [-1,+1]
-    assert(bits == 5 || bits == 7); // 5 for RPC hits from CPPF, 7 for CSC LCTs
+  inline int    calc_theta_int(double theta, int endcap) {  // theta in deg, endcap [-1,+1]
     theta = (endcap == -1) ? (180. - theta) : theta;
-    theta = (theta - 8.5) * pow(2, bits) / (45.0-8.5);
+    theta = (theta - 8.5) * 128./(45.0-8.5);
     int theta_int = static_cast<int>(std::round(theta));
     return theta_int;
   }
 
+  inline int    calc_theta_int_rpc(double theta, int endcap) {  // theta in deg, endcap [-1,+1]
+    theta = (endcap == -1) ? (180. - theta) : theta;
+    theta = (theta - 8.5) * (128./4.)/(45.0-8.5);  // 4x coarser resolution
+    int theta_int = static_cast<int>(std::round(theta));
+    return theta_int;
```

Comment:
If I'm thinking about this right, this is incorrect ... and was incorrect in the previous version as well.  The integer theta scale should be the same as for CSCs, but the resolution is 4 x coarser.  So what should be returned is theta_int * 4.